### DISCUSSION TITLE
Is RabbitMQ-server affected by the Log4j remote code execution vulnerability?

### DISCUSSION THREAD

**Discussion by iigx** [rabbitmq/rabbitmq-server#3886] [Other] [answered]
Hello,
I wonder if RabbitMQ is affected by the Log4j remote code execution vulnerability ?
Currently I'm using docker version of RabbitMQ 3.8.9 with below command:
>docker pull rabbitmq:3.8.9-management

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
RabbitMQ is not a Java or JVM-based tool. It does not use log4j and neither do our Java clients.

**Comment by LindaLawton** [comment]
@michaelklishin  thanks for your response.  What about the usage of Erlang?  Does that use log4j?

**Reply by lukebakken** [reply]
Erlang is its own virtual machine (i.e. **not** Java). The source code is here if you'd like to verify - https://github.com/erlang/otp

**Reply by LindaLawton** [reply]
@lukebakken Thank you for taking the time to respond.   I didn't think it was an issue but I needed something concreate to go back to my client with.

**Comment by lukebakken** [comment, marked as answer]
https://blog.rabbitmq.com/posts/2021/12/rabbitmq-not-affected-by-log4j-vulnerability/
